web/src/LogLine.scss

```diff
@@ -40,10 +46,17 @@
   }
 }
 
-.LogLine:not(.is-buildEvent) + .LogLine.is-buildEvent,
-.LogLine.is-buildEvent + .LogLine:not(.is-buildEvent) {
+.LogLine:not(.is-buildEvent) + .LogLine.is-buildEvent {
   margin-top: 8px;
 }
+.LogLine.is-buildEvent + .LogLine:not(.is-buildEvent) > .LogLine-content,
+.LogLine.is-buildEvent + .LogLine:not(.is-buildEvent) > .logLinePrefix {
+  padding-top: 8px;
+}
+.LogLine:not(.is-buildEvent) + .LogLine.is-buildEvent.is-sticky,
+.LogLine.is-buildEvent.is-sticky:first-child {
+  top: -8px; // For position:sticky with padding on LogPane
```

Comment:
definitely not a merge-blocker, but i still see the 8px of padding when i scroll all the way to the top of the log pane. 

![2022-02-25 12 21 46](https://user-images.githubusercontent.com/23283986/155759361-a282f839-411f-4591-8385-294b53dda9e4.gif)

fiddling around in the browser, i was able to remove it by setting the log pane padding to `0 0 8px 0` and then this can be set to `0`.

Reply:
oh that's intentional! that's what it looks like at head (go here https://storybook.tilt.dev/?path=/story/new-ui-overviewresourcepane--ten-resources-with-log-store and click "all resources")

Reply:
ahh, interesting! nevermind then.